### ISSUE TITLE
Unable to ignore `.. code-block:`

### ISSUE THREAD

**Issue by phlax** [myint/rstcheck#79]
Currently `code-block` checking doesnt work when (some?) sphinx args are supplied in the rst (see #62 )

I then tried to disable code-block by adding it to `ignore_directives` but that didnt seem to work either

**Comment by Cielquan** [comment]
When I use the code example from #62 the ignore flag does work for me as well as setting it in `.rstcheck.cfg` when I have sphinx installed.

```rst
.. code-block:: yaml
   :caption: **caption**
   :emphasize-lines:
```

```shell
$ rstcheck test.rst
test.rst:1: (ERROR/3) Error in "code-block" directive:
```
```shell
$ rstcheck test.rst --ignore-directives code-block
```

Without sphinx the directive is not getting ignored.

**Comment by Cielquan** [comment]
I think the root cause is here: https://github.com/myint/rstcheck/blob/master/rstcheck/__init__.py#L143-L148

**Comment by Cielquan** [comment]
The issue here is also much deeper.

**Comment by Cielquan** [comment]
Ok now this issue is sustainably fixed in #94.

This is btw not related to #62. Both are two separate issues underneath.
